msp/cache/cache.go

```diff
@@ -83,14 +83,16 @@ func (c *cachedMSP) DeserializeIdentity(serializedIdentity []byte) (msp.Identity
 }
 
 func (c *cachedMSP) Setup(config *pmsp.MSPConfig) error {
-	c.cleanCache()
+	if err := c.cleanCache(); err != nil {
```

Comment:
instead of doing this maybe simply make the cleanCache method not return an error? I don't see how it can fail.

Reply:
Yes this is redundant